pages/expense.js

```diff
@@ -96,11 +96,13 @@ class ExpensePage extends React.Component {
                   ← <FormattedMessage id="expenses.viewAll" defaultMessage="View All Expenses" />
                 </StyledButton>
               </Link>
-              <Link route="createExpense" params={{ collectiveSlug: collective.slug }}>
-                <StyledButton my={1} mx={3} data-cy="submit-expense-btn">
-                  <FormattedMessage id="expenses.sendAnotherExpense" defaultMessage="Submit Another Expense" />
-                </StyledButton>
-              </Link>
+              {collective.isActive && (
```

Comment:
`isActive` is not the right field for that. This field will be false if the collective is pending an approval from the host, but users should still be able to submit expenses in that case. You may want to look at the `isArchived` field instead.